.gitignore

```diff
@@ -4,7 +4,6 @@
 *.pyc
 __pycache__/
 local_settings.py
-migrations
```

Comment:
What was the goal of adding migrations to being tracked by git?

Reply:
[See here.](https://stackoverflow.com/a/28035246/2129808)

Migrations aren't necessarily trival. (Right now they are, but that could change in the future.)

Reply:
In my experience working with Django, it is better to leave them out for build time's sake and for keeping minor changes out of the migrations log.

Yes Django official docs say to keep them, but even with a project with 12 different models it's still not necessary.

If we want to add them later when/if models become non-trivial let's do that. For now though I'd rather not have 40 changes to iterate through on fresh builds.

Reply:
Further, in my development of projects I've found that nuking models and starting over is sometimes necessary. Unless you want to spend ~40 minutes figuring out how and where to manually edit migrations files.

Merge conflicts pretty much only happen when numerous people are working on models at the same time on a local branch (VSCode remote sessions come to mind). Or when you change a critical attribute, such as key name

Reply:
Alight, I'll take your word for it as I haven't used Django much.

(To be honest, I included them because I was having a weird issue where `manage.py migrate` wouldn't do anything. I'll take a look; maybe `migrations/` was still in .dockerignore at that point.)

Reply:
Fixed in 2565086.